### ISSUE TITLE
[BUG] src/blocks seems to be missing when I install

### ISSUE THREAD

**Issue by Oosik** [infinum/eightshift-boilerplate#229]
I'm unable to install. I'm getting errors after installation. The install appears successful but when I try to activate the theme or run the `wp boilerplate` command as instructed in the success message I get:

```
Fatal error: Uncaught Eightshift_Libs\Exception\Invalid_Block: Global blocks settings manifest.json is missing on this location: /var/www/fr/client/wp-content/themes/test/src/blocks/manifest.json. in /var/www/fr/client/wp-content/themes/test/vendor/infinum/eightshift-libs/src/exception/class-invalid-block.php:101
Stack trace:
#0 /var/www/fr/client/wp-content/themes/test/vendor/infinum/eightshift-libs/src/blocks/class-blocks.php(472): Eightshift_Libs\Exception\Invalid_Block::missing_settings_manifest_exception('/var/www/fr/cli...')
#1 /var/www/fr/client/wp-content/themes/test/vendor/infinum/eightshift-libs/src/blocks/class-blocks.php(117): Eightshift_Libs\Blocks\Blocks->get_settings()
#2 /var/www/fr/client/wp-content/themes/test/vendor/infinum/eightshift-libs/src/blocks/class-blocks.php(104): Eightshift_Libs\Blocks\Blocks->get_colors_from_settings()
#3 /var/www/fr/client/wp-includes/class-wp-hook.php(287): Eightshift_Libs\Blocks\Blocks->change_editor_color_palette('')
#4 /var/www/fr/client/wp-includes/class-w in /var/www/fr/client/wp-content/themes/test/vendor/infinum/eightshift-libs/src/exception/class-invalid-block.php on line 101
Error: There has been a critical error on your website.Learn more about debugging in WordPress. There has been a critical error on your website.
```
After the installer runs, I do not have a src/blocks directory. I only have admin, media, and menu directories with class-main and class-config in my src directory.

Additionally when running `wp boilerplate setup_theme` from inside themes/test/ I get the error above but also 

`Error: 'boilerplate' is not a registered wp command. See 'wp help' for available commands.`

and when I run `npm start` from themes/test/

```
/var/www/fr/client/wp-content/themes/test/node_modules/webpack-cli/bin/cli.js:93
				throw err;
				^

Error: Cannot find module '/var/www/fr/client/wp-content/themes/test/src/blocks/manifest.json'
```

## Steps to Reproduce

1. npx create-wp-project
2. wp theme activate test

## Environment

* PHP version: 7.3.24
* Development environment: just a vhost on my local machine, no VM
* Server type: Apache
* Operating System and version: Linux Mint 19 Tara

## Update
I just tried in one of the default sites in VVV and was able to switch the theme but still getting errors when trying to use `wp boilerplate` or `npm start` or viewing the site in the browser:

```
? Please enter your project name (e.g. Your Project Name): test
? Your dev url (e.g. dev.wordpress.com): one.wordpress.test
? Your project's description: test

Summary: 
- projectName: test
- url: one.wordpress.test
- description: test
- package: test
- namespace: Test
- prefix: TES
- env: TES_ENV
- projectPrefix: tes
? Looks good? Yes

✔ 1. Cloning repo
✔ 2. Installing Node dependencies
✔ 3. Replacing theme info
✔ 4. Installing Composer dependencies
✔ 5. Cleaning up
----------------
Success!!!

Please do the following steps manually to complete the setup:
1. Activate your new theme by running wp theme activate test
2. Run wp boilerplate --help to see what's possible using our WP-CLI commands.
3. If you can't decide what to do, we recommend running wp boilerplate setup_theme inside your new theme folder.

Please read the documentation https://infinum.github.io/eightshift-docs/ if you run into any issues or if you have any questions.

Best of luck!
----------------
vagrant@vvv:/srv/www/wordpress-one/public_html/wp-content/themes$ wp theme activate test
Success: Switched to 'test' theme.
vagrant@vvv:/srv/www/wordpress-one/public_html/wp-content/themes$ cd test/
vagrant@vvv:/srv/www/wordpress-one/public_html/wp-content/themes/test$ wp boilerplate setup_theme
Fatal error: Uncaught Eightshift_Libs\Exception\Invalid_Block: Global blocks settings manifest.json is missing on this location: /srv/www/wordpress-one/public_html/wp-content/themes/test/src/blocks/manifest.json. in /srv/www/wordpress-one/public_html/wp-content/themes/test/vendor/infinum/eightshift-libs/src/exception/class-invalid-block.php:101
Stack trace:
#0 /srv/www/wordpress-one/public_html/wp-content/themes/test/vendor/infinum/eightshift-libs/src/blocks/class-blocks.php(472): Eightshift_Libs\Exception\Invalid_Block::missing_settings_manifest_exception('/srv/www/wordpr...')
#1 /srv/www/wordpress-one/public_html/wp-content/themes/test/vendor/infinum/eightshift-libs/src/blocks/class-blocks.php(117): Eightshift_Libs\Blocks\Blocks->get_settings()
#2 /srv/www/wordpress-one/public_html/wp-content/themes/test/vendor/infinum/eightshift-libs/src/blocks/class-blocks.php(104): Eightshift_Libs\Blocks\Blocks->get_colors_from_settings()
#3 /srv/www/wordpress-one/public_html/wp-includes/class-wp-hook.php(287): Eightshift_Libs\Blocks in /srv/www/wordpress-one/public_html/wp-content/themes/test/vendor/infinum/eightshift-libs/src/exception/class-invalid-block.php on line 101
vagrant@vvv:/srv/www/wordpress-one/public_html/wp-content/themes/test$ npm start

> @eightshift/test@4.3.0 start /srv/www/wordpress-one/public_html/wp-content/themes/test
> webpack --progress --watch --display-error-details --display-reasons --mode development

/srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/webpack-cli/bin/cli.js:93
				throw err;
				^

Error: Cannot find module '/srv/www/wordpress-one/public_html/wp-content/themes/test/src/blocks/manifest.json'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (/srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/webpack-cli/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at module.exports (/srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/@eightshift/frontend-libs/webpack/base.js:96:28)
    at module.exports (/srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/@eightshift/frontend-libs/webpack/index.js:34:33)
    at module.exports (/srv/www/wordpress-one/public_html/wp-content/themes/test/webpack.config.js:23:69)
    at handleFunction (/srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/webpack-cli/bin/utils/prepareOptions.js:21:13)
    at prepareOptions (/srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/webpack-cli/bin/utils/prepareOptions.js:9:5)
    at requireConfig (/srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/webpack-cli/bin/utils/convert-argv.js:117:14)
    at /srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/webpack-cli/bin/utils/convert-argv.js:123:17
    at Array.forEach (<anonymous>)
    at module.exports (/srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/webpack-cli/bin/utils/convert-argv.js:121:15)
    at yargs.parse (/srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/webpack-cli/bin/cli.js:71:45)
    at Object.parse (/srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/webpack-cli/node_modules/yargs/yargs.js:567:18)
    at /srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/webpack-cli/bin/cli.js:49:8
    at Object.<anonymous> (/srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/webpack-cli/bin/cli.js:366:3)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/srv/www/wordpress-one/public_html/wp-content/themes/test/node_modules/webpack/bin/webpack.js:156:2)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @eightshift/test@4.3.0 start: `webpack --progress --watch --display-error-details --display-reasons --mode development`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @eightshift/test@4.3.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2020-11-24T22_05_46_659Z-debug.log
vagrant@vvv:/srv/www/wordpress-one/public_html/wp-content/themes/test$ 
```

## Update 2
npx create-wp-project@1.0.7 seems to work 
npx create-wp-project@2.0.0 fails during install (@ npm run build)
npx create-wp-project@2.0.4 fails same error `Fatal error: Uncaught Eightshift_Libs\Exception\Invalid_Block: Global blocks settings manifest.json is missing on this location: /srv/www/wordpress-one/public_html/wp-content/themes/test/src/blocks/manifest.json. in /srv/www/wordpress-one/public_html/wp-content/themes/test/vendor/infinum/eightshift-libs/src/exception/class-invalid-block.php:101`

**Comment by IvanGrginovInf** [comment]
> npx create-wp-project@1.0.7 seems to work

Hi @Oosik for now if possible please use `npx create-wp-project@1.0.7` for now. We're currently in the process of releasing a new version of boilerplate which will fix all these issues.

**Comment by Oosik** [comment]
FYI: still broken in 2.0.7 release a few hours ago. eagerly awaiting a fix. Thanks!

**Comment by iruzevic** [comment]
Sorry about that, we fixed it now. Until we release version 5 you can use this command to setup:

`npx create-wp-project --eightshiftBoilerplateBranch="release/5.0.0"`

**Comment by Oosik** [comment]
> npx create-wp-project --eightshiftBoilerplateBranch="release/5.0.0"

when I use this and activating the theme - I'm getting:
```
/var/www/fr/client/wp-content/themes/test  $ wp boilerplate setup_theme
Error: 'boilerplate' is not a registered wp command. See 'wp help' for available commands.
```
Is there a separate install process for the wpcli command that I'm missing?

**Comment by Oosik** [comment]
> npx create-wp-project --eightshiftBoilerplateBranch="release/5.0.0"
when I use this and activating the theme - I'm getting:
/var/www/fr/client/wp-content/themes/test  $ wp boilerplate setup_theme
Error: 'boilerplate' is not a registered wp command. See 'wp help' for available commands.
Is there a separate install process for the wpcli command that I'm missing?

Okay so it looks like the paths are getting a little messed up. I looked in functions.php and found this `use TestVendor\EightshiftLibs\Cli\Cli;` I changed it to `use EightshiftLibs\Cli\Cli;` and was able to use the boilerplate wpcli command.

Running the theme setup was successful but when I viewed the site I was getting similar errors: **Fatal error: Uncaught Error: Class 'TestVendor\EightshiftLibs\Helpers\Components'**

So I ran a find/replace in the theme directory. that got me to the next path error where it couldn't find the manifest class. I changed `namespace EightshiftBoilerplate\Manifest;` to `namespace Test\Manifest` which fixed that error but now I'm getting **Fatal error: Uncaught Error: Class 'EightshiftBoilerplate\Config\Config'** ...

basically, everytime I fix one another one crops up. What I've noticed so far:

1) in composer
```
"extra": {
		"imposter": {
			"namespace": "TestVendor"
		}
	}
```

2) in themes/test/src -- classes. The namespace seems wrong
3) need to fix TestVendor\EightshiftLibs

I'm still getting 'class not found' errors, and since it seems pretty widespread I didn't want to continue down this path as it seems like it's just going to cause issues down the line. Hopefully there is a quick fix. Thanks!
